kubedifflib/_diff.py

```diff
@@ -21,6 +21,12 @@
   iter_files,
 )
 
+def mask(x):
+  """Turn a string into an equal-length string of asterisks"""
+  try:
+    return len(x) * '*'
+  except TypeError: # not a string - perhaps None - just return it as-is
+    return x
```

Comment:
Not sure what (other than string or None) can be passed in here, but maybe return a fixed number of asterisks rather than the un-masked value?  Just don't know if there might be possible non-string secret data passed in...